website/src/roadmap/euphoria.md

```diff
@@ -23,7 +23,74 @@ limitations under the License.
 Easy to use Java 8 DSL for the Beam Java SDK. Provides a high-level abstraction of Beam transformations, which is both easy to read and write. Can be used as a complement to existing Beam pipelines (convertible back and forth). You can have a glimpse of the API at [WordCount example]({{ site.baseurl
 }}/documentation/sdks/java/euphoria/#wordcount-example).
 
-This roadmap is in progress. In the meantime, here are some resources:
-
 - JIRA: [dsl-euphoria](https://issues.apache.org/jira/browse/BEAM-4366?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20dsl-euphoria) / [BEAM-3900](https://issues.apache.org/jira/browse/BEAM-3900)
-- Contact: [David Moravek](mailto:david.moravek@gmail.com)
\ No newline at end of file
+- Contact: [David Moravek](mailto:dmvk@apache.org)
```

Comment:
Can we have documentation link here? Currently we can get there through `WordCount example`, but that is not really straightforward.

Reply:
👍 makes sense